README.md

```diff
@@ -98,6 +98,10 @@ protocols:
         exitcode?: number
           # Mocked exitcode of the command.
           # Default: 0
+        filetree: string
```

Comment:
Should we call this `mockedFiles` instead of `filetree`?

Reply:
i have no preference, i'll change it